### DISCUSSION TITLE
Changing and saving camera resolution

### DISCUSSION THREAD

**Discussion by AleksanderKruecken** [monkeymademe/CamUI#83] [General]
Hi,

I have gone to Control camera > Camera capture settings > Changed resolution.

Do I have to save this settings or is it enough that i pressed the radio button for new resolution? 
Will the resolution stay after service restart?

There is a button Load/Save settings and option 'Save current settings as new file'.  Must I choose this option?

In camera info there is still old resolution showed. I can click and change the mode in camera info, but I'm confused where do I change camera resolution - is it in the Control camera tab or is it in the Camera info tab? What do I have to do to save the new resolution?

Kind regards,
Aleksander

**Comment by monkeymademe** [comment]
Control camera > Camera capture settings > Changed resolution

This changes the output resolution for the feed... If you use the Load/Save settings option you can save this config to a file and when you reboot it will be persistent because its saved to the conifg. If you want to update the setting you can just use the same filename when saving and it will overwrite the config with the new setting. Its not great but yeah.

In Camera info you are seeing the sensor mode. This is a separate value that determines what pixels are taken from sensor. Its like a crop on the sensor. (v3 models are less extreme but the v2 cameras have quite a few sensor modes) So you can have a sensor mode set to 640x480 and output it to 1920x1080 like a image resize... You can also have 3280x2464 and output it to 640x480... 

If that makes sense?

Oh and sensor mode is also saved when you save the config.

**Reply by monkeymademe** [reply]
If the config is set to a camera if you reboot it 'should' reload that config but if you change the camera physically from one model to another. There is a last_connected_camera config which resets and tells the system there is no config so use default. Then you will have to reload the config from the menu again. 

But yeah thats only when you start fussing with the actual camera and should technically not happen often

**Reply by AleksanderKruecken** [reply]
Thank you for explanation. Now it makes sense to me.

**Comment by monkeymademe** [comment]
I created a feature request for adding some info for this - https://github.com/monkeymademe/picamera2-WebUI/issues/85
